tests/test_standard_tags.py

```diff
@@ -38,7 +38,6 @@ def test_method_trace(self):
 @released(dotnet="2.13.0", golang="1.40.0", java="0.107.1", nodejs="3.0.0")
 @released(php="0.76.0", python="1.6.0rc1.dev", ruby="?")
 @rfc("https://datadoghq.atlassian.net/wiki/spaces/APS/pages/2490990623/QueryString+-+Sensitive+Data+Obfuscation")
-@bug(weblog_variant="spring-boot-undertow", reason="APMJAVA-877")
```

Comment:
you probably need to set this (where `x.y.z` is the version that fixes the bug) : 

```python
@released(java=
        {
            "spring-boot-undertow": "x.y.z",  # see APMJAVA-877
            "*": "0.107.1"
        }
)
```

Reply:
I'm not sure there was a bug, or I couldn't identify a version that fixed anything. The test apparently started working after some changes in system-tests themselves...